examples/training_scripts/resnet_cifar10/resnet_cifar10.py

```diff
@@ -50,7 +50,6 @@ def resnet_cifar10_default_params(params=None):
         'epochs': 27,
         'grace_period': 1,
         'reduction_factor': 3,
-        'max_resource_attr': 'epochs',
```

Comment:
If we move nas201 in `benchmark/definitions/` I think we should also move those definitions but this could be done in a separate PR.

Reply:
Exactly, I have that on my list, to separate code from meta-data for all the benchmarks.